src/testing/jest/install-dependencies.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# Get the directory where this script is located
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# Change to the directory where the script is located so that it can be called from anywhere
+cd "$SCRIPT_DIR"
```

Comment:
sometimes bash confuses me, but didn't we already `cd` to `$0` on line 6?

Reply:
In line 6, we're inside of a subshell, so that `cd` doesn't get persisted to our current directory state. Added a comment in bfd16e4a3 to clarify

Reply:
right, so it's like

1. dirname gives us the relative path from the calling location to the script's directory
2. cd-ing to that then puts us in that directory
3. doing `pwd` gives us the full path to the script dir

makes sense when you think it through haha